frontend/package.json

```diff
@@ -8,7 +8,10 @@
         "apexcharts": "^3.26.1",
         "axios": "^0.21.1",
         "cors": "^2.8.5",
+        "enzyme": "^3.11.0",
+        "enzyme-adapter-react-16": "^1.15.6",
         "html2canvas": "^1.0.0-rc.7",
+        "jest-cli": "^20.0.4",
```

Comment:
Don't forget to update the BOM in Google Drive if you add new dependencies here @SaiVarunVaranasi